activerecord/test/cases/associations/required_test.rb

```diff
@@ -22,30 +22,6 @@ class Child < ActiveRecord::Base
     @connection.drop_table "children", if_exists: true
   end
 
-  test "belongs_to associations are not required by default" do
```

Comment:
Would be valid change this PR to instead of remove these tests, make them relate with that new configurable behaviour for `belongs_to`, or it would be better close that PR?